.github/pull-request-template.md

```diff
@@ -1,40 +1,41 @@
 ## **Description**
-_Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:_
-_1. What is the reason for the change?_
-_2. What is the improvement/solution?_
+
+<!--
+Write a short description of the changes included in this pull request, also include relevant motivation and context. Have in mind the following questions:
+1. What is the reason for the change?
+2. What is the improvement/solution?
+-->
+
+## **Related issues**
+
+Fixes: #
 
 ## **Manual testing steps**
 
-_1. Step1:_
-_2. Step2:_
-_3. ..._
+1. Go to this page...
+2.
+3.
```

Comment:
Removes italics and allows the engineer to remove "this page..." and start writing instructions instead of removing the underscores and Step1: etc